core/os_shared.h

```diff
@@ -314,6 +314,39 @@ char *
 get_application_pid(void);
 char *
 get_application_name(void);
+
+/* DR_API EXPORT BEGIN */
+/**
+ * Encodings of an application's argument.
+ */
+typedef enum {
+    DR_APP_ARG_CSTR_COMPAT,
```

Comment:
These values should have doxygen comments.

Reply:
Done.